Remove botched DataTable.row_headers deprecation code in 0.12.15

I've just updated from 0.12.14 to 0.12.15

I'm getting the following failure from my server-side code:

    E           AttributeError: unexpected attribute 'row_headers' to DataTable, similar attributes are index_header

OK I think, possibly this attribute went away.  Lets see how bad things look if I just don't specify it at all.

But when I remove it from my figure I get the following error on the client

    connection.js:189 [bokeh] Failed to repull session Error: property DataTable.row_headers wasn't declared

--- a/bokeh/models/widgets/tables.py
+++ b/bokeh/models/widgets/tables.py
@@ -7,7 +7,6 @@ from ...core.enums import DateFormat, FontStyle, NumeralLanguage, TextAlign, Rou
 from ...core.has_props import abstract
 from ...core.properties import Bool, Color, Either, Enum, Float, Instance, Int, List, Override, String
 from ...model import Model
-from ...util.deprecation import deprecated
 
 from ..sources import DataSource, CDSView
 
@@ -500,23 +499,6 @@ class DataTable(TableWidget):
     of data.
 
     '''
-
-    @property
-    def row_header(self):
-        deprecated("DataTable.row_header property is deprecated, use DataTable.index_position instead"\
-                   "In the future, accessing DataTable.row_header will result in an AttributeError.")
-        return self.index_position is not None
-
-    @row_header.setter
-    def row_header(self, val):
-        deprecated("DataTable.row_header property is deprecated, use DataTable.index_position instead."\
-                   "In the future, accessing DataTable.row_header will result in an AttributeError.")
-        if val is True:
-            self.index_position = 0
-        elif val is False:
-            self.index_position = None
-        else:
-            raise ValueError("Expected True or False for DataTable.row_header, got %s" % val)
 
     columns = List(Instance(TableColumn), help="""
     The list of child column widgets.

